utoipa-gen/src/path/response.rs

```diff
@@ -62,8 +62,18 @@ impl Parse for ResponseValue<'_> {
             match attribute_name {
                 "status" => {
                     response.status_code =
-                        parse_utils::parse_next(input, || input.parse::<LitInt>())?
-                            .base10_parse()?;
+                        parse_utils::parse_next(input, || {
+                            // Allows us to keep using integer literals to maintain compatibility
+                            // And also strings for "default" "4XX" etc.
+                            let lookahead = input.lookahead1();
+                            if lookahead.peek(LitInt) {
+                                input.parse::<LitInt>()?.base10_parse()
+                            } else if lookahead.peek(LitStr) {
+                                Ok(input.parse::<LitStr>()?.value())
+                            } else {
+                                Err(lookahead.error())
+                            }
+                        })?
```

Comment:
While this is okay, I'm also wondering that this does not limit the the user of defining a value as "sflksfjl" just random string, and when such is defined user will only see error when he opens the browser with the documentaiton.

Reply:
Ok I have changed it to either accept integer literals, or strings from a validated list

Reply:
Awesome :+1: